apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/cql/ColumnUtils.java

```diff
@@ -22,7 +22,8 @@ public class ColumnUtils {
 
   private static final Pattern PATTERN_DOUBLE_QUOTE = Pattern.compile("\"", Pattern.LITERAL);
   private static final String ESCAPED_DOUBLE_QUOTE = Matcher.quoteReplacement("\"\"");
-  private static final Pattern UNQUOTED_IDENTIFIER = Pattern.compile("[a-z][a-z0-9_]*");
+  private static final Pattern UNQUOTED_IDENTIFIER =
```

Comment:
Maybe add a note that this needs to support indexing (brackets), with an example, in javadoc comment.